wrappers/rust/icicle-cuda-runtime/src/device_context.rs

```diff
@@ -1,27 +1,42 @@
 use crate::memory::CudaMemPool;
 use crate::stream::CudaStream;
 
-/// Properties of the device used in icicle functions.
+pub const DEFAULT_DEVICE_ID: usize = 0;
+
+use crate::device::get_device;
+
+/// Properties of the device used in Icicle functions.
 #[repr(C)]
 #[derive(Debug, Clone)]
 pub struct DeviceContext<'a> {
-    /// Stream to use. Default value: 0.
+    /// Stream to use. Default value: 0. //TODO: multiple streams per device ?
```

Comment:
wdym by multiple streams per device?

Reply:
array of streams vs array of DeviceContext